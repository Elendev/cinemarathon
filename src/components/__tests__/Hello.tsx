import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Hello from '../Hello';

test('Ensure Hello display correctly', () => {
    const hello = renderer.create(
        <Hello/>,
    );
    expect(hello.toJSON()).toMatchSnapshot();
});
