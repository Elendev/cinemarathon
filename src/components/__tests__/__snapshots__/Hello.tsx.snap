// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ensure Hello display correctly 1`] = `
<h1>
  Hello from Cinemarathon !
</h1>
`;
